xarray/tests/test_computation.py

```diff
@@ -789,6 +811,84 @@ def func(x):
     assert_identical(expected, actual)
 
 
+def test_corr(da):
+
+    # other: select missaligned data, and smooth it to dampen the correlation with self.
+    da_smooth = da.isel(time=range(2, 20)).rolling(time=3, center=True).mean(dim="time")
+
+    da = da.isel(time=range(0, 18))
+
+    def select_pts(array):
+        return array.sel(a=1, x=2)
+
+    # Test #1: Misaligned 1-D dataarrays with missing values
+    ts1 = select_pts(da.copy())
+    ts2 = select_pts(da_smooth.copy())
+
+    def pd_corr(ts1, ts2):
+        """Ensure the ts are aligned and missing values ignored"""
+        # ts1,ts2 = xr.align(ts1,ts2)
+        valid_values = ts1.notnull() & ts2.notnull()
+
+        ts1 = ts1.where(valid_values, drop=True)
+        ts2 = ts2.where(valid_values, drop=True)
+
+        return ts1.to_series().corr(ts2.to_series())
+
+    expected = pd_corr(ts1, ts2)
+    actual = xr.corr(ts1, ts2)
+    assert np.allclose(expected, actual)
+
+    # Test #2: Misaligned N-D dataarrays with missing values
+    actual_ND = xr.corr(da, da_smooth, dim="time")
+    actual = select_pts(actual_ND)
+    np.allclose(expected, actual)
```

Comment:
Same for all these `np.allclose`: it returns a bool, so we need an assert

Reply:
Sure, thanks! Added `assert statements` to all of those tests.